Assets/AppCenter/Plugins/AppCenterSDK/Auth/Blank/AuthInternal.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#if (!UNITY_IOS && !UNITY_ANDROID && !UNITY_WSA_10_0) || UNITY_EDITOR
```

Comment:
Does `#if !(UNITY_IOS || UNITY_ANDROID || UNITY_WSA_10_0) || UNITY_EDITOR` look more readable? 😄

Reply:
This is copied from other files. Better to keep it and change it all together. Thanks for the suggestion.